data/mods/blazemod/blaze_blob.json

```diff
@@ -584,20 +593,29 @@
   {
     "id": "gray_tank",
     "type": "CONTAINER",
-    "name": "gray tank",
-    "name_plural": "gray tanks",
+    "name": "gray cocoon",
+    "name_plural": "gray cocoons",
     "description": "While the blob is very eager to be fed, it's not as enthusiastic about giving up liquids.  A few alterations are necessary.",
     "weight": 19120,
-    "volume": 30,
+    "volume": 40,
     "price": 100,
     "to_hit": -2,
     "material": [ "flesh" ],
     "symbol": ";",
     "color": "light_gray",
-    "contains": 100,
+    "contains": 120,
     "seals": true,
     "watertight": true
   },
+  {
+    "id": "gray_tank_large",
+    "type": "CONTAINER",
+    "name": "gray tank",
+    "name_plural": "gray tanks",
+    "copy-from": "gray_tank",
+    "volume": 400,
```

Comment:
Needs to mass more than a gray_tank - at least twice as much, since it has roughly 10x the volume and 4x the surface area.